requirements.txt

```diff
@@ -1,4 +1,6 @@
 libsass
 git+https://github.com/edx/XBlock.git#egg=XBlock
 git+https://github.com/edx/xblock-utils.git#egg=xblock-utils
--e .
\ No newline at end of file
+pep8
+pylint
+-e .
```

Comment:
Let's reorder these before they get out of hand.
1. Named python packages
2. Packages available and fetched from Github
3. `-e .`

It would be nice to have a comment before each section and an empty line in between sections.